apps/example/components/InputDisplay.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+
+import { CustomInputProps } from "@premieroctet/next-admin";
+
+type Props = CustomInputProps;
+
+const InputDisplay = ({ value, name }: Props) => {
```

Comment:
I think we should have an example that allows changing the value so that we show how to handle value modification. Maybe a custom date picker ?